src/plugins/sys/components/picklist-picker-field/index.jsx

```diff
@@ -1,116 +1,44 @@
-import PropTypes from 'prop-types';
 import React from 'react';
-import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
-import Code from '@gdbots/schemas/gdbots/pbjx/enums/Code';
-import createDelegateFactory from '@triniti/app/createDelegateFactory';
-import Exception from '@gdbots/common/Exception';
-import Message from '@gdbots/pbj/Message';
-import {
-  Alert,
-  FormGroup,
-  FormText,
-  Label,
-  Select,
-  StatusMessage,
-} from '@triniti/admin-ui-plugin/components';
-import { STATUS_FAILED, STATUS_REJECTED } from '@triniti/app/constants';
+import { FormGroup } from '@triniti/admin-ui-plugin/components';
 
-import delegateFactory from './delegate';
-import selector from './selector';
+import PicklistPicker from '../picklist-picker/index';
```

Comment:
@cohencodes use the absolute path and never import index: 

`import PicklistPicker from '@triniti/cms/plugins/sys/components/picklist-picker';`